hack/validate-prow-job-semantics.py

```diff
@@ -167,6 +167,13 @@ def validate_names(path, data):
                             logger.error("%s: job %s branches with underscores are not allowed: %s", path, job["name"], branch_name)
                     branch = make_regex_filename_label(job["branches"][0])
 
+                # This is an undocumented and temporary hack to allow some custom jobs to use targets like "images"
```

Comment:
@droslean i added this as a quick workaround for the errors in the prow-config-semantics job due to the custom jobs targeting `images`:

```
ERROR: ./ci-operator/jobs/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master-postsubmits.yaml: file contains jobs for more than one repo
ERROR: ./ci-operator/jobs/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master-presubmits.yaml: ci-operator job pull-ci-openshift-qe-baremetal-qe-infra-master-images-multi should be named one of ['pull-ci-openshift-qe-baremetal-qe-infra-master-images'] ```

Let me know if this can be enough as a temporary turnaround while you continue the automation work, or you prefer something else.

Reply:
cc @danilo-gemoli